src/main.rs

```diff
@@ -71,6 +75,15 @@ struct RunArgs {
     name: String,
 }
 
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "reset")]
+/// Resets a single exercise
```

Comment:
This could be more detailed, e.g. `Resets a single exercise by using "git stash"`